packages/api/tests/controllers/grantCycle.test.js

```diff
@@ -0,0 +1,60 @@
+const request = require('supertest');
+const app = require('../../server');
+const db = require('../../models');
+
+// TODO make test to make sure relations work with Nomination as expected
+
+describe('POST Grant Cycle Endpoint', () => {
+  let body = {};
+
+  beforeAll(() => {
+    body = {
+      name: "test grant",
+      openedOn: new Date("2019-01-01"),
+      closedOn: new Date("2020-01-01")
```

Comment:
When we receive request, most dates will come in string format. Is Express automatically converting these to JS `Date`s?

Reply:
good question. In this case, I assumed the input will be a string in YYYY-MM-DD format. I think the js Date object can handle variations of these. I'll post a question about this